packages/wallet/src/TxBuilder/types.ts

```diff
@@ -32,13 +32,23 @@ export class OutputValidationTokenBundleSizeError extends CustomError {
   }
 }
 
+export class TxBuilderDisabled extends CustomError {
```

Comment:
```suggestion
export class TxAlreadySubmitted extends CustomError {
```

Reply:
Sounds much better, thanks! I will also add the `Error` suffix: `TxAlreadySubmittedError`

Reply:
Updated in: [d0bfbb2](https://github.com/input-output-hk/cardano-js-sdk/pull/434/commits/d0bfbb2dbae618fec7ec774c5aa6b12578e92f34)